coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -54,13 +54,11 @@ def _extract_doc_comment_continuous(content, line, column, markers):
 
     The property of the continuous layout is that the each-line-marker and the
     end-marker do equal. Documentation is extracted until no further marker is
-    found. Applies e.g. for doxygen style python documentation:
+    found. Applies e.g. for doxygen style python documentation::
```

Comment:
The first line of this commit message is "Use correct RST syntax".  Currently it is using markdown syntax.